README.md

```diff
@@ -82,8 +82,8 @@ if you allow the `process_subway.py` to fetch data from Overpass API. Here are t
 Summary information about all metro networks that are monitored is gathered in the
 [Google Spreadsheet](https://docs.google.com/spreadsheets/d/1SEW1-NiNOnA2qDwievcxYV1FOaQl1mb1fdeyqAxHu3k).
 
-Not so regular updates of validation results are available at
-[this website](https://alexey-zakharenkov.github.io/subways/rapid/).
+Regular updates of validation results are available at
```

Comment:
```suggestion
Regular (weekly) updates of validation results are available at
```

But not insising on it, it might be not so interesting.

Reply:
Added link to OSM wiki